formats/hocon/src/test/kotlin/kotlinx/serialization/hocon/HoconPolymorphismTest.kt

```diff
@@ -0,0 +1,154 @@
+package kotlinx.serialization.hocon
+
+import com.typesafe.config.ConfigFactory
+import kotlinx.serialization.SerialName
```

Comment:
We usually enable star imports in Editor-Code Style-Kotlin because they're almost every time from one of the library's packages